src/kili/services/copy_project/__init__.py

```diff
@@ -119,3 +157,154 @@ def _copy_quality_settings(self, new_project_id: str, src_project: Dict) -> None
             use_honeypot=src_project["useHoneyPot"],
             review_coverage=src_project["reviewCoverage"],
         )
+
+    def _copy_assets(self, from_project_id: str, new_project_id: str, batch_size: int = 200):
+        """
+        Copy assets from a project to another.
+
+        Fetches assets by batch since `content` urls expire.
+        """
+        nb_assets_to_copy = self.count_assets(project_id=from_project_id)
```

Comment:
Clients project can be quite big, I would add some logging here for the copy of members, assets and labels.
like 'copying members...', 'copying assets...', 'copying labels...'

Reply:
I added tqdm bars for assets and labels, and some logging for each step of the copying process

Reply:
I tested it, It is good, we now have a better expectation of the time needed to execute the function.
Can you also add a tqdm progress bar when copying members then ?

Reply:
And also, when copying labels, I have a weird behavior, the progress bar does not update, then update almost to the end and then change it total to 1.
See here:
I give you access to the project
[Enregistrement de l’écran 2022-12-28 à 16.10.12.zip](https://github.com/kili-technology/kili-python-sdk/files/10314865/Enregistrement.de.l.ecran.2022-12-28.a.16.10.12.zip)

Reply:
In the project, there are 600 labels, 4 of them have the status REVIEWED.

Since I batch labels by status for before calling `append_labels`, it is normal that the progress bar is stuck at 0 and then jumps at 596.

I don't know why the progress bar changes its total to 1 though... I will investigate. Thanks for the feedback

Reply:
Ah yes it is because you update the progress bar after each (labelType, modelName) and you disable tqdm for the append_labels mutation.
I would rather show tqdm inside the append_labels query because if a client has a project with only 10 000 labels, he will be stuck on the 'copying labels... 0% during a while...'